documentation/architecture/openspec/project.md

```diff
@@ -1,31 +1,68 @@
 # Project Context
 
 ## Purpose
-[Describe your project's purpose and goals]
+Mimeogram is a command-line tool for **exchanging collections of files with Large Language Models (LLMs)**. It bundles multiple files into a single clipboard-ready document while preserving directory structure and metadata, making it ideal for code reviews, project sharing, and LLM interactions. It supports interactive reviews, clipboard integration, and path protection.
 
 ## Tech Stack
-- [List your primary technologies]
-- [e.g., TypeScript, React, Node.js]
+- **Language**: Python 3.10+
+- **Build System**: Hatch
+- **Packaging**: PyInstaller (for standalone executable)
+- **Key Libraries**:
+  - `accretive` (accretive data structures)
+  - `aiofiles` (async file I/O)
+  - `detextive` (text detection)
+  - `httpx` (HTTP client)
+  - `icecream-truck` (debugging)
+  - `patiencediff` (diff generation)
+  - `pyperclip` (clipboard integration)
+  - `rich` (terminal formatting)
+  - `tiktoken` (token counting)
+  - `tomli` (TOML parsing)
+  - `tyro` (CLI argument parsing)
+  - `appcore` (application core utilities)
 
 ## Project Conventions
 
 ### Code Style
-[Describe your code style preferences, formatting rules, and naming conventions]
+- **Line Length**: 79 characters
+- **Indentation**: 4 spaces
+- **Type Hinting**: Strict type checking with `pyright` (reportUnknownArgumentType, etc. are true).
+- **Linters**:
+  - `ruff` (general linting)
+  - `isort` (import sorting)
+  - `vibelinter` (custom linter)
```

Comment:
We do not need to mention `vibelinter`. It's name is likely to change in the longer term.